web/src/lib/managers/timeline-manager/types.ts

```diff
@@ -19,6 +19,7 @@ export type TimelineAsset = {
   thumbhash: string | null;
   localDateTime: TimelinePlainDateTime;
   fileCreatedAt: TimelinePlainDateTime;
+  originalFileName: string;
```

Comment:
this is wrong, we don't select or return the original file name for the time bucket queries and this type is now incorrect.